### ISSUE TITLE
Stop resetting wa_col to aquifer_water_baseline

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/ctsm#596]
As @bishtgautam pointed out in #524 , we really shouldn't be resetting wa_col to aquifer_water_baseline here:

https://github.com/ESCOMP/ctsm/blob/7755baf3f729ff9adb0e39088c02113bbc9ebb58/src/biogeophys/BalanceCheckMod.F90#L197-L206

@swensosc and I talked quite a bit about this when he was working on his groundwater_irrigation changes, and we weren't sure of the implications of completely removing this – but he did enclose the update in a conditional on use_aquifer_layer, so it isn't done for the more modern CLM50 physics. However, in addition to being generally problematic, I'm realizing that this resetting is problematic for water tracers. So we really do need to remove it.

**Comment by billsacks** [comment]
@swensosc I'm tentatively assigning this to you to think about what we should do here - i.e., if we can simply remove it, or if we need to do something else so that CLM45 physics continues to work with this removed. Feel free to reassign if you don't feel you're the right one to tackle this.

**Comment by billsacks** [comment]
@bishtgautam feels it's safe to simply remove this block of code (see #524).

**Comment by wwieder** [comment]
Should we close these long-open water isotope issues with a won't fix, as there's no plan or funding to do this work?

**Comment by billsacks** [comment]
I'd vote for keeping these open as low priority. I am periodically in discussions with Bette about getting funding for someone to resume this work. But if the issues are cluttering things up for your process, maybe we can find some other way to note what still needs to be done.

**Comment by ekluzek** [comment]
I wonder if another label for these could be used? Something like "no-funding"? We could close them, but if something changes we could then find them and open them again. This would also be applied to other issues that similarly need some resources before they could be pursued. You could also leave them open, but I'd side with closing them until the status changes. What are thoughts on that idea? @wwieder and @billsacks?

**Comment by billsacks** [comment]
First, I just read back through this issue, and I think it's only tangentially related to water isotopes. So I think what we do with this issue doesn't necessarily relate strongly to what we do for the water isotope issues.

For the core water isotope issues in the water isotope project (https://github.com/ESCOMP/CTSM/projects/9), my leaning is still towards keeping them open but possibly adding a label - like Erik's "no-funding" or maybe something more general like "stalled" - that could be used to remove these from searches. For this and similar stalled projects, I feel like this will make it easier for someone to hit the ground running once there is funding / time / interest in picking it back up. For example, I put some thought into the arrangement of the cards in that project board - though I realize that it wouldn't be too hard to record the arrangement somewhere so it could be recreated by reopening the issues and redoing the arrangement - or maybe we could turn off the automation on the project board before closing the issues so that the closed issues maintain their position in the columns?

I don't feel strongly about this as long as there is some way to recreate the current state if you go ahead and close them. But I guess I'd first want to ask what the problem is that we're trying to solve here by closing the issues, to see if there's an easy workaround that would apply to this and other cases, like adding "-label:stalled" to the issue searches that are turning up these issues. That said, if the workaround would be annoying or need to be done a lot, I can definitely see the argument for closing these issues for now and reopening them later.

**tl;dr: I still lean towards keeping issues open in cases like these, possibly adding a new label; but I don't feel strongly about this so would be happy deferring to you, @wwieder and @ekluzek .**

**Comment by billsacks** [comment]
As noted in #4123 , I think we'll just prevent users from running with water tracers along with an aquifer layer, so we don't need to resolve this issue for the sake of water tracers.

So I'm tentatively closing this as a wontfix; we can reopen later if needed.
